.github/workflows/deploy-image.yml

```
@@ -0,0 +1,34 @@
+jobs:
+      push-app-image:
+        runs-on: ubuntu-latest
+        defaults:
+          run:
+            working-directory: './apps/api'
+        steps:
+          - name: 'Checkout GitHub Action'
+            uses: actions/checkout@main
+
+          - name: 'Login to GitHub Container Registry'
+            uses: docker/login-action@v1
```

actionlint (1.7.4)

the runner of "docker/login-action@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)